### ISSUE TITLE
Location for enquiries and awards

### ISSUE THREAD

**Issue by yolile** [open-contracting/standard#749]
Colombia has to publish the location for make enquiries and also the location where the awards it is going to be. 
Example:

[LUGAR_ACLARACION] : AREA DE CONTRATACION Av 5 Calles 13 y 14 Secretaria General Segundo Piso
[LUGAR_ADJUDICACION]:  SALA DE JUNTAS  Av 5 Calles 13 y 14

The proposal is to add the fields 
- tender/enquiriesAddress and 
- award/awardAddress 

 of type **Address**

**Comment by duncandewhurst** [comment]
@yolile it might be better to use `tender/enquiriesAddress` and `tender/awardAddress`, in keeping with the terminology used in the location extension and the organization block.

**Comment by yolile** [comment]
Colombia created the extensions for this: 
https://github.com/sdd1982/expressionAddress
https://github.com/sdd1982/expressionEmail
https://github.com/sdd1982/enquiriesAddress
https://github.com/sdd1982/awardAddress

**Comment by jpmckinney** [comment]
In the EU's Tenders Electronic Daily, the contact points for enquiries and review are modelled as organizations. So, in the EU profile, to provide the addresses/contact points for enquiries and expressions of interest, we use the `parties` array, and add to that party's `roles` codes like 'processContactPoint' or 'reviewContactPoint'. 

https://github.com/open-contracting-extensions/european-union/issues/9
https://github.com/open-contracting-extensions/european-union/issues/25

However, if jurisdictions like Colombia have only an address/email without other organization details, then we might need to consider a different common model.

**Comment by jpmckinney** [comment]
The EU profile also needs to model "place for opening of tenders", for which we might consider a common/similar solution as in Colombia.

**Comment by jpmckinney** [comment]
For the EU profile, we're proposing adding a `tender.bidOpening` object, with fields fo address, date, description.

**Comment by jpmckinney** [comment]
cc @ColinMaudry

**Comment by jpmckinney** [comment]
Here is the extension: https://github.com/open-contracting-extensions/ocds_bidOpening_extension (I will review and merge the PR shortly)

**Comment by jpmckinney** [comment]
For award address, this is modelled as part of the information about bid opening more broadly (which would include the date and other details). The extension is now in the Explorer: https://extensions.open-contracting.org/en/extensions/bidOpening/master/

For enquiries address, this is typically modelled as the address of the party that handles enquiries (usually the procuring entity). For the EU, we added a [processContactPoint](https://extensions.open-contracting.org/en/extensions/eu/master/codelists/#+partyRole.csv) party role. So, I don't think we need a new field – at most, a new party role.

Closing as use cases served by new extension and existing model.
